app/assets/javascripts/protocol_form.js.coffee

```diff
@@ -56,108 +55,36 @@ $(document).ready ->
-      add_and_check_visual_error_on_field_change($(higher_level_of_privacy_dropdown))
-
-    $(access_required_dropdown).on 'change', (e) ->
-      add_and_check_visual_error_on_field_change($(access_required_dropdown))
-
-  #### This was written for an edge case in admin/portal.
-  #### When you go from a virgin project (selected_for_epic = nil/ never been a study)
-  #### to a study, the Epic Box should be editable instead of only displaying the epic box data.
-
-  if $('#study_can_edit_admin_study').val() == "can_edit_study"
-    $('#actions input[type="submit"]').on 'click', (e) ->
-      if $('input[name=\'study[selected_for_epic]\']:checked').val() == 'true'
-        if $(certificate_of_confidence_dropdown).val() == ''
-          epic_box_alert_message()
-          add_and_check_visual_error_on_submit($(certificate_of_confidence_dropdown))
-          return false
-        if $(certificate_of_confidence_dropdown).val() == 'false'
-          if $(higher_level_of_privacy_dropdown).val() == ''
-            epic_box_alert_message()
-            add_and_check_visual_error_on_submit($(higher_level_of_privacy_dropdown))
-            return false
-          if $(higher_level_of_privacy_dropdown).val() == 'true'
-            if $(access_required_dropdown).val() == ''
-              epic_box_alert_message()
-              add_and_check_visual_error_on_submit($(access_required_dropdown))
-              return false
-            if $(access_required_dropdown).val() == 'false'
-              if $(epic_inbasket_dropdown).val() == '' ||$(research_active_dropdown).val() == '' || $(restrict_sending_dropdown).val() == ''
-                epic_box_alert_message()
-                add_and_check_visual_error_on_submit($(epic_inbasket_dropdown))
-                add_and_check_visual_error_on_submit(research_active_dropdown)
-                add_and_check_visual_error_on_submit($(restrict_sending_dropdown))
-                return false
-          else if $(higher_level_of_privacy_dropdown).val() == 'false'
-            if $(epic_inbasket_dropdown).val() == '' ||$(research_active_dropdown).val() == '' || $(restrict_sending_dropdown).val() == ''
-              epic_box_alert_message()
-              add_and_check_visual_error_on_submit($(epic_inbasket_dropdown))
-              add_and_check_visual_error_on_submit(research_active_dropdown)
-              add_and_check_visual_error_on_submit($(restrict_sending_dropdown))
-              return false
-  ######## End of send to epic study question logic ##############
+  determine_study_type = (answers) ->
+    array_values = new Array()
+    for k,v of answers
+      array_values.push(v)
+    nil_value = $.inArray('', array_values) > -1
+    if array_values[0] == 'true' || nil_value == false
+      $.ajax
+        type: 'POST'
+        data: answers
+        url: "/study_type/determine_study_type_note"
+        success: ->
+          $('#study_type_note').show()
```

Comment:
What happens if this errors out? Should we do some error handling here on AJAX error?